projects/pluto/README.md

```diff
@@ -0,0 +1,37 @@
+# Pluto
+
+The service is a Spring Boot service playing role of a lightweight API gateway using Spring Cloud Gateway.
```

Comment:
Or 'is a Spring Boot application' to avoid 'service' repetition?

Reply:
updated